app/views/admin/budgets/_drafting.html.erb

```diff
@@ -0,0 +1,16 @@
+<% if can? :publish, @budget %>
+  <%= link_to t("admin.budgets.edit.publish"),
+              publish_admin_budget_path(@budget),
+              method: :patch, class: "button float-right",
+              data: { confirm: t("admin.actions.confirm") } %>
+<% end %>
+
+<%= link_to budget_path(@budget), class: "button hollow float-right", target: "_blank" do %>
+  <span class="far fa-eye"></span><%= t("admin.budgets.edit.preview") %>
```

Comment:
When it's already published, it's a bit strange to **pre**view a budget :thinking:. In that case, we would be viewing it and not previewing it.